File: tests/eval_c/084.c
#include <assert.h>
#include <string.h>

void f084(const char *s) {
    char t[2];
    strncpy(t, s, 1);
    t[1] = '\0';

    assert(strcmp(t, "a") == 0);
    assert(strlen(s) == 1);
}

File: benchmarks/oopsla25/subjects/084.py
import sys

def f084(s: str):
  t = s[0:1]
  assert t == "a"
  assert len(s) == 1

if __name__ == '__main__':
  my_string = ""
  if len(sys.argv) == 1:
    my_string = sys.stdin.read()
  else:
    fd = open(sys.argv[1], 'r')
    my_string = fd.read()
    fd.close()
  
  f084(my_string)
